Formula/svgo.rb

```diff
@@ -0,0 +1,21 @@
+require "language/node"
+
+class Svgo < Formula
+  desc "Nodejs-based tool for optimizing SVG vector graphics files"
+  homepage "https://github.com/svg/svgo"
+  url "https://github.com/svg/svgo/archive/v0.7.2.tar.gz"
+  sha256 "93b36477366e6b48b9bc3f9086232ae7fc16b87fa36acc1d524720ea58123fcf"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    cp test_fixtures("test.svg"), testpath
```

Comment:
I'd rather copy fixtures before using them. We know what should happen but not what will happen, and that's why we need tests. Yes, I'm aware of sandboxing, but `--no-sandbox` is still supported.